packages/bitcoin/src/bip322/bip322-utils.ts

```diff
@@ -0,0 +1,88 @@
+// Compatible package that isn't tiny-secp256k1
```

Comment:
While we are moving these files, it seems like a good time to break them up. We shouldn't have huge `utils.ts` files, functions should be separated into their own files